lib/utils/addEntries.js

```diff
@@ -1,51 +1,53 @@
 'use strict';
 
 /* eslint-disable
+  no-shadow,
   no-param-reassign,
+  array-bracket-spacing,
   space-before-function-paren
 */
 const createDomain = require('./createDomain');
 
-function addEntries (webpackOptions, devServerOptions, server) {
-  if (devServerOptions.inline !== false) {
+function addEntries (config, options, server) {
+  if (options.inline !== false) {
     // we're stubbing the app in this method as it's static and doesn't require
     // a server to be supplied. createDomain requires an app with the
     // address() signature.
     const app = server || {
       address() {
-        return { port: devServerOptions.port };
+        return { port: options.port };
       }
     };
 
-    const domain = createDomain(devServerOptions, app);
-    const devClient = [`${require.resolve('../../client/')}?${domain}`];
+    const domain = createDomain(options, app);
+    const entries = [ `${require.resolve('../../client/')}?${domain}` ];
```

Comment:
Looks your have ` ` here, maybe better run prettier?

Reply:
The space is intenentional `[ a, b ]` here, prettier does hopefully leave it :)

Reply:
@michael-ciniawsky hm, we should integrate prettier here :confused:

Reply:
Yep, prettier will be included with `webpack-defaults` in `webpack-dev-server >= v4.0.0`, it should still leave whitespace in this case since it is more readable

Reply:
@michael-ciniawsky :+1: